### PR TITLE
Resolve PE bugs

### PR DESCRIPTION
**Pull request by jordankanghm**
Resolved the following issues.

Events with same name but different fields are not allowed
-Indicated in UG that event names are identity fields.

Description and Location cannot take brackets
-Changed the regex for Description and Location such that they can accept all characters

"eaddv -> vedit -> eremovev" and "eaddv -> vedit -> elistv" does not work as intended
-Indicated in UG that volunteer names are identity fields, hence vedit will not be able to change the names of volunteers.

Corrected the error message for ecreate to show a valid example

Edit: Created a new PR cos i fked up the other branch lol

### CODE REVIEW

File: src/main/java/seedu/address/model/event/Description.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ public class Description {
      * otherwise " " (a blank string) becomes a valid input.
      * The expression allows for multiple words to be inputted.
      */
-    public static final String VALIDATION_REGEX = "[A-Za-z0-9 _.,!\"'/$]+";
+    public static final String VALIDATION_REGEX = "\\S.*";
```
Comment (zacwong2151) [inline src/main/java/seedu/address/model/event/Description.java:19]: what regex is this?

Comment (zacwong2151) [reply src/main/java/seedu/address/model/event/Description.java:19]: like what does it accept

**Review by zacwong2151 — APPROVED** [review]
lgtm

**Review by brein62 — APPROVED** [review]
lookin good, good lookin

File: src/main/java/seedu/address/model/event/Description.java (hunk 1 above)
Comment (brein62) [reply src/main/java/seedu/address/model/event/Description.java:19]: `\\S`: anything non-whitespace
`.`: any character

Comment (zacwong2151) [reply src/main/java/seedu/address/model/event/Description.java:19]: legend

**Pull request merged**